lib/common_test/src/ct_snmp.erl

```diff
@@ -206,6 +217,11 @@ start(Config, MgrAgentConfName) ->
 %%% to configure the snmp application with parameters such as <code>config</code>,
 %%% <code>mibs</code>, <code>net_if</code>, etc. The values will be merged
 %%% with (and possibly override) default values set by <code>ct_snmp</code>.
+%%% @end
+-spec start(Config, MgrAgentConfName, SnmpAppConfName) -> ok when
+      Config :: agent_config(),
```

Comment:
This should not be <code>agent_config()</code>. It is the <code>Config</code> variable from the test case, defined as [{Key,Value}], where Key::atom(), Value::term().